src/libImaging/Quant.c

```diff
@@ -522,57 +522,53 @@ split(BoxNode *node) {
     }
 #ifdef DEBUG
     printf("along axis %d\n", axis + 1);
-    {
```

Comment:
This is required for MS compiler, since variables could be defined only in the beginning of the scope

Reply:
Ok, I've removed the commit.